src/NuGet.Core/NuGet.Protocol/Utility/LocalFolderUtility.cs

```diff
@@ -949,6 +949,47 @@ public static IEnumerable<LocalPackageInfo> GetPackagesV3(string root, string id
             yield break;
         }
 
+
+        public static IEnumerable<string> ResolvePackageFromPath(string packagePath)
```

Comment:
Add a /// summary here to make it clear to anyone else using this that the method is applying wild cards